packages/react-widgets/src/models/utils.js

```diff
@@ -27,7 +27,7 @@ export function formatTableNameWithFilters(props) {
   const whereClause = _filtersToSQL(filters, filtersLogicalOperator);
 
   const formattedSourceData =
-    source.type === MAP_TYPES.QUERY ? `(${data.replace(';', '')})` : data;
+    source.type === MAP_TYPES.QUERY ? `(${data.replace(';', '')}) foo` : data;
```

Comment:
I'd use another alias for the subquery. If I read a query like this, I'm going to think it's like a test.

Reply:
The alias won't be used, so I don't want the user to think that the alias is useful.